password_security/__manifest__.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2015 LasLabs Inc.
+# Copyright 2018 Modoolar <info@modoolar.com>.
+# Copyright 2019 initOS GmbH
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+{
+    "name": "Password Security",
+    "summary": "Allow admin to set password security requirements.",
+    "version": "18.0.2.0.0",
```

Comment:
It is just small correction.
```suggestion
    "version": "18.0.1.0.0",
```